overlord/snapstate/snapstate.go

```diff
@@ -876,3 +876,37 @@ func GadgetInfo(s *state.State) (*snap.Info, error) {
 
 	return nil, state.ErrNoState
 }
+
+// InstallMany installs everything from the given list of names.
+// Note that the state must be locked by the caller.
+func InstallMany(st *state.State, names []string, userID int) ([]string, []*state.TaskSet, error) {
```

Comment:
Can we use this to simplify the ubuntu-core initial installation (in the future)?